src/site/content/en/fast/codelab-use-lazysizes-to-lazyload-images/index.md

```diff
@@ -91,7 +90,7 @@ Lastly, it's a good idea to use Lighthouse to verify these changes. Lighthouse's
 "Defer offscreen images" performance audit will indicate if you've forgotten to
 add lazy loading to any offscreen images.
 
-- Preview the site with the **Show** button.
+{% Instruction 'preview' %}
 
 - Run the Lighthouse performance audit (Lighthouse > Options > Performance) on
```

Comment:
Should this and the following line be replaced by the performance-audit short code?

Reply:
yep :D

Reply:
Fixed in new PR